src/globus_sdk/transport/encoders.py

```diff
@@ -31,6 +31,7 @@ def encode(
                 "Either manually encode the data or use `encoding=form|json` to "
                 "correctly format this data."
             )
+        data = self._prepare_data(data)
```

Comment:
I did a self-review (on my way to hitting merge) and this jumped out as an unwanted line. Let me fix to remove.
(I think I was going to pull this out into a separate block in my first steps, and then I realized it wasn't necessary. So this is just a stray edit.)